hermes/Earthfile

```diff
@@ -13,6 +13,13 @@ builder:
     RUN mkdir /wasm
     COPY --dir ../wasm+wasi-src/wasi /wasm/wasi
 
+# Expands `wasmtime::bindgen!` macro into the `bindings.rs` file
+bindings-expand:
+    FROM +builder
+
+    RUN cargo expand -p hermes --lib runtime_extensions::bindings > bindings.rs
+    SAVE ARTIFACT bindings.rs AS LOCAL bindings.rs
+
```

Comment:
I would suggest that this is a resposibility of whoever changes the bindings, WDYT?

Reply:
Did not get what do you mean.
I have added this just for the convenience of the local development, so you can alway easily take a look how it is expanded